code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -206,8 +206,14 @@
 	STR.can_hold = typecacheof(list(/obj/item/clothing/suit/space/syndicate, /obj/item/clothing/head/helmet/space/syndicate))
 
 /obj/item/storage/box/syndie_kit/space/PopulateContents()
-	new /obj/item/clothing/suit/space/syndicate/black/red(src) // Black and red is so in right now
-	new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
+	switch (pickweight(list("goon" = 1, "normal" = 1)))

```

Comment:
No need for this, just put it in an `if(prob(50))` statement.

Reply:
if i'm gonna be honest im not sure where to put that

Reply:
`if(prob(50)
    do_stuff()
else
    something_else()`